juju/python-libjuju
# Bugfix none type

#### Description

*\<Please add why this change is needed and what it does.\>*
Fix for #1028 
The logger is not passed to calls of `utils.run_with_interupt` from model.py and controller.py. The default value is `None`, so when it is later used we get the error. 
*\<Fixes:  \>*


#### QA Steps
Added a `raise Exception("my random exception")` to the `Next` method of `ModelSummeryWatcherFacade`. This is hit by the `run_with_interupt` in controller and we can observe that when we run the file `modelsummaries.py`, if the log arguments added in this PR are not there then we get the None Type error and if they are there, we do not.

Jira: JUJU-5610
